Lib/glyphsLib/builder/color_layers.py

```diff
@@ -141,6 +143,38 @@ def to_ufo_color_layers(self):
                 paint = dict(
                     Format=PaintFormat.PaintSolid, Alpha=a, PaletteIndex=paletteIndex
                 )
+            elif (
+                "strokeColor" in attributes
+                or "strokeWidth" in attributes
+                or not attributes
+            ):
+                try:
+                    import pathops
+                except ImportError as ex:
+                    raise RuntimeError(
+                        f"Stroked color layer path in '{glyph.name}' requires 'pathops' module."
+                    ) from ex
+
+                pos = attributes.get("strokePos", 0)
+                if pos:
+                    raise NotImplementedError(
+                        f"Unsupported color layer path attribute 'strokePos' in '{glyph.name}'"
```

Comment:
what does strokePos mean/do?

Reply:
It controls on which side of the path the stroke is drawn:
<img width="537" alt="image" src="https://user-images.githubusercontent.com/93914/143483481-a5dc1324-7d03-4eb0-b615-3956ca24ba2d.png">

Reply:
Exporting font with stroke path set to anything other than the default currently crashes Glyphs anyway.